packages/@secretlint/secretlint-rule-pattern0/package-lock.json

```diff
@@ -0,0 +1,1631 @@
+{
```

Comment:
This repository use [Yarn v1](https://classic.yarnpkg.com/).

https://github.com/secretlint/secretlint/blob/master/CONTRIBUTING.md

`package-lock.json` will not needed.